packages/beacon-node/src/chain/archiveStore/archiveStore.ts

```diff
@@ -32,7 +32,6 @@ export class ArchiveStore {
   private archiveMode: ArchiveMode;
   private jobQueue: JobItemQueue<[CheckpointWithHex], void>;
 
-  private prevFinalized: CheckpointWithHex;
```

Comment:
@nazarhussain do you know why we have this? seems like was last modified in https://github.com/ChainSafe/lodestar/pull/7591 but looks like your PR didn't really change behavior either, but maybe you know why we kept it around

Reply:
> added to keep track for logging purpose

in that case there should be a explicit comment, having unused code can be really confusing when looking at the implementation so should get rid of it now

Reply:
Sure you can remove it now.